entries/s/sendwyre.com.json

```
@@ -0,0 +1,17 @@
+{
+  "Wyre": {
+    "domain": "sendwyre.com",
+    "additional-domains": [
+      "dash.sendwyre.com"
+    ],
+    "tfa": [
+      "sms",
+      "custom-software"
```

Looks like TOTP is used.
```suggestion
      "totp"
```